src/lib/__mocks__/mockApiRequests.ts

```diff
@@ -77,19 +83,116 @@ const mockedProduct = {
   id: 'prod-io',
 };
 
+const mockedSubProduct = {
+  title: 'Premium',
+  id: 'prod-io-premium',
+  parent: 'prod-io',
+};
+
+const mockedParties: Array<SelfcareParty> = [
+  {
+    partyId: '0',
+    originId: '0',
+    externalId: '0',

```

Comment:
use different values for these 3 fields in order to be make more strong test if checking the values